pkg/features/kube_features.go

```diff
@@ -1281,4 +1287,6 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	// features that enable backwards compatibility but are scheduled to be removed
 	// ...
 	HPAScaleToZero: {Default: false, PreRelease: featuregate.Alpha},
+
+	NamespaceIndex: {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
We can probably go directly to beta - we won't get anything from using alpha disabled by default.